### ISSUE TITLE
Volume and lag

### ISSUE THREAD

**Issue by redFrik** [supercollider/supercollider#3326]
the updateSynth in Volume does not respect lag time.

```supercollider
s.boot
{SinOsc.ar([400,404],0,0.5)}.play
s.volume.lag= 5
s.volume.volume = -59  //these 
s.volume.volume = -0.01
s.volume.volume = -59
s.volume.volume = -0.01
s.volume.volume = -inf  //here problem starts
s.volume.volume = 0  //the volume synth is released by the updateSynth method
s.volume.volume = -inf
s.volume.volume = 0
```

a __half__ fix would be to simply add lag to release...

```supercollider
if(ampSynth.notNil) { ampSynth.release; ampSynth = nil }
if(ampSynth.notNil) { ampSynth.release(lag); ampSynth = nil }
```

then at least 50% of the lagged volume changes will work as expected (the ones where the synth is already running).

reference: http://new-supercollider-mailing-lists-forums-use-these.2681727.n2.nabble.com/Programmatically-changing-the-master-volume-td7611143.html

**Comment by patrickdupuis** [comment]
@redFrik I think #3332 fixes this.

**Comment by redFrik** [comment]
elegant fix @patrickdupuis !  #3332 seems to work fine.  thanks.

**Comment by brianlheim** [comment]
Closed by #3332.
